package.json

```diff
@@ -27,10 +27,16 @@
   "homepage": "https://github.com/AmicaleCORE/Karl-API#readme",
   "devDependencies": {
     "eslint": "^8.36.0",
-    "nodemon": "^2.0.22"
+    "nodemon": "^2.0.22",
+    "sqlite3": "^5.1.6"
```

Comment:
You need to install mariadb and mysql as well.
```
npm i --save mysql2 mariadb
```
These modules are supported by Sequelize and help you to interact with few more databases.